tests/integration/db.py

```diff
@@ -133,21 +172,19 @@ def get_month(delta=0):
     return new
 
 
-def hasher(uaid):
-    # type: (str) -> str
+def hasher(uaid: str) -> str:
     """Hashes a key using a key_hash if present"""
     if key_hash:
         return generate_hash(key_hash, bytes(uaid, "utf-8"))
     return uaid
 
 
 def create_message_table_ddb(
-    tablename,  # type: str
-    read_throughput=5,  # type: int
-    write_throughput=5,  # type: int
-    boto_resource=None,  # type: DynamoDBResource
-):
-    # type: (...) -> Any  # noqa
+    boto_resource: DynamoDBResource,
```

Comment:
Not a huge fan of changing the order of parameters, but this is a limited use library. 

Still, it's a good idea to keep order and only append new args to the end of a given function declaration.

Reply:
I 100% agree, but made a judgment, very much with the understanding that the impact was contained. 

**Background**
Previously, the `boto_resource` was declared as an Optional parameter in many places which defaulted to `None` in the method declaration. This causes mypy to complain, i think rightly so, when we proceed to use `boto_resource` without checking if it is `None`. 

Option 1 was to add null checks, which added unnecessary complexity in my opinion. So I opted for Option 2, to remove the optionality for `boto_resource`, but then I ran into the limitation that non-default method arguments need to be listed before defaulted arguments. I propagated this change in order across methods to be consistent, but perhaps that's too much.